source/extensions/quic_listeners/quiche/platform/BUILD

```diff
@@ -9,11 +9,20 @@ load(
 
 envoy_package()
 
-# Build targets in this package are part of the QUICHE platform implementation,
-# are are not to be consumed or referenced directly by other Envoy code. The
-# only consumers should be build rules under @com_googlesource_quiche//..., and
-# tests. In a monorepo, this would be enforced via visibility attribute, but
+# Build targets in this package are part of the QUICHE platform implementation.
+# These implementations are the infrastructure building block for QUIC. They are
+# used in 2 different ways:
+#
+# Most of them are not to be consumed or referenced directly by other Envoy code.
+# Their only consumers should be build rules under @com_googlesource_quiche//...,
+# and tests. In a monorepo, this would be enforced via visibility attribute, but
 # Bazel does not support limiting visibility to specific external dependencies.
+#
+# Very few of them are used by Envoy to interact with QUIC. They are used as a shim
+# to match a non-virtualized API required by the external Quiche implementation.
```

Comment:
Note also that we are deliberately omitting new unit tests for the platform impl, as the code is tested from quic. Also point how to find the coverage for those tests.